### PR TITLE
Making coveralls work again

### PR DESCRIPTION
**Pull request by lorenzo**

### CODE REVIEW

**Comment by ADmad** [comment]
:+1:  I made a small change to how xdebug.ini is removed.

File: .travis.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ script:
 
   - sh -c "if [ '$PHPCS' = '1' ]; then vendor/bin/phpcs -p --extensions=php --standard=vendor/cakephp/cakephp-codesniffer/CakePHP ./src ./tests; fi"
 
-  - sh -c "if [ '$COVERALLS' = '1' ]; then phpunit --coverage-clover build/logs/clover.xml; fi"
+  - sh -c "if [ '$COVERALLS' = '1' ]; then phpdbg -qrr vendor/bin/phpunit --coverage-clover build/logs/clover.xml || true; fi"
```
Comment (markstory) [inline .travis.yml]: Why phpdbg?

Comment (lorenzo) [inline .travis.yml]: More than twice as fast: 17 minutes vs 8 minutes with phpdbg. It also has a more accurate report of covered lines

Comment (markstory) [inline .travis.yml]: wow.

Comment (josegonzalez) [inline .travis.yml]: @lorenzo how did you find `phpdbg`? Sounds like an interesting blog post...

**Pull request merged**